src/client/mapblock_mesh.cpp

```diff
@@ -963,8 +961,7 @@ static void updateFastFaceRow(
 		face_dir_corrected = next_face_dir_corrected;
 		std::memcpy(lights, next_lights, ARRLEN(lights) * sizeof(u16));
```

Comment:
```suggestion
		std::memcpy(lights, next_lights, sizeof(lights));
```
And the same on line 924